.storybook/preview.js

```diff
@@ -2,6 +2,7 @@ import { addDecorator, addParameters } from '@storybook/html';
 import { withA11y } from '@storybook/addon-a11y';
 import { withPaddings } from 'storybook-addon-paddings';
 import * as colors from '../src/design-tokens/colors.yml';
+import { breakpointViewports } from './helpers/breakpoint-viewports';
```

Comment:
Can we include this logic alongside the setting itself? I find it odd to have _just_ these nine lines of configuration in a separate file.

Reply:
Personal preference and something we've been doing recently in other projects - trying to keep small, focused modules of code to avoid having files grow too large. I'm fine with moving this back into `preview.js` if you like that better.

Reply:
I have done this, and also updated comments.